engine/src/main/java/org/hibernate/validator/cfg/defs/ScriptAssertDef.java

```diff
@@ -32,4 +32,9 @@ public ScriptAssertDef alias(String alias) {
 		addParameter( "alias", alias );
 		return this;
 	}
+
+	public ScriptAssertDef reportOn(String reportOn) {
+		addParameter( "reportOn", reportOn );
+		return this;
+	}
```

Comment:
Is the `ScriptAssertDef` class tested somewhere? If so, it would be nice to also test the `reportOn` option.